.travis.yml

```diff
@@ -65,19 +60,42 @@ jobs:
         # case that the travis log doesn't produce any output for more than 10 minutes
         - while sleep 540; do echo "==== Still running (travis, don't kill me) ===="; done &
       script:
-        - docker run -e CI -e TRAVIS_JOB_ID="$TRAVIS_JOB_ID" -e TRAVIS_BRANCH="$TRAVIS_BRANCH" p4a /bin/sh -c "$COMMAND"
+        - >
+          docker run
+          -e CI
+          -e TRAVIS_JOB_ID="$TRAVIS_JOB_ID"
+          -e TRAVIS_BRANCH="$TRAVIS_BRANCH"
```

Comment:
Just for your info, you can simplify to `-e TRAVIS_BRANCH` since variables are the same you can skip the `=` part. Just like we did for `-e CI`